biz.aQute.launcher/src/aQute/launcher/plugin/ProjectLauncherImpl.java

```diff
@@ -343,6 +343,19 @@ public Jar executable() throws Exception {
 				"SHA-1", "MD-5"
 			});
 		jar.setManifest(m);
+
+		String moduleInstruction = getProject().getProperty(Constants.MODULE);
+		if (moduleInstruction != null) {
```

Comment:
What is the point of this code here? Hasn't the project already been built before the launcher runs? Why rebuild? and isn't that risky?